db/migrate/20190612174127_add_checked_terms_and_conditions_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddCheckedTermsAndConditionsToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :checked_terms_and_conditions, :bool
```

Comment:
Side question: does this mean that all existing users will have to sign T&C again?

Reply:
We had a more implicit situation in the past, so based on how we were interpreting things we were technically okay but we wanted to take this as an opportunity to be more explicit about it before proceeding.

Reply:
This should probably have a default value of `false`.